### PR TITLE
added my details to team.md

### PR DESCRIPTION
**Pull request by samueladeniyi**

### CODE REVIEW

**Review by shekhar10feb — APPROVED** [review]

**Review by ni3-kdm — APPROVED** [review]

File: docs/TEAM.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
-| ![avatar](https://github.com/RafaelBatman55.png?size=80) | [Rafael Rodrigues](https://github.com/RafaelBatman55)        | Less than 1 yr of experience From small city in Rio de Janeiro State - Braz.I'm very excited to learn and helf in this project!                                                                                                                                                                                                                                                                            |
+| ![avatar](https://github.com/RafaelBatman55.png?size=80)                                 | [Rafael Rodrigues](https://github.com/RafaelBatman55)        | Less than 1 yr of experience From small city in Rio de Janeiro State - Braz.I'm very excited to learn and helf in this project!                                                                                                                                                                                                                                                                            |
```
Comment (RafaelBatman55) [inline docs/TEAM.md:12]: Why did you add this kind of space after my avatar? Is it necessary to works better?

Comment (samueladeniyi) [reply docs/TEAM.md:12]: Please, I am using prettier extension for code formatting,  I did not edit your code

Comment (arthurbuhl) [reply docs/TEAM.md:12]: Please disable the prettier extension for this workspace in VS Code, revert the changes and add your local changes again.

Comment (samueladeniyi) [reply docs/TEAM.md:12]: Ok, I will do it

Comment (RafaelBatman55) [reply docs/TEAM.md:12]: No problem @samueladeniyi!
